setup.py

```diff
@@ -48,6 +49,9 @@ def get_static_files(path):
     install_requires.remove('python-daemon<3.0')
     install_requires.append('sphinx>=1.4.4')  # Value mirrored in doc/conf.py
 
+if sys.version_info < (3, 4):
+    install_requires.append('enum34>1.1.0')
```

Comment:
I'm not strongly against this. But it will break for some users who don't install the package the normal way.

Reply:
@dlstadther, what do you think? Maybe it's time to request users to always install through a package manager, hopefully not many will oppose this.

Reply:
How does one use luigi without either PyPi or running `setup.py`?

So long as people can appropriately use a released version from PyPi or bleeding edge through manual install, I'm :+1:

Reply:
Well, you could just download the source filles and set your `PYTHONPATH=luigi` and do `python  luigi/.xxx.py` and get going. For the server you still needed tornado.

Reply:
I don't think people will complain having to install with a package manager or manually setup.py install/develop